artifact/gem5art/artifact/artifact.py

```diff
@@ -202,28 +197,56 @@ def registerArtifact(cls,
         data['is_zipped'] = is_zipped
         data['md5sum'] = md5sum
         data['url'] = url
-        data['supported_gem5_versions'] = supported_gem5_versions
+        data['supported_gem5_versions'] = supported_gem5_versions[:]
         data['version'] = version
 
-        for k, v in kwargs.items():
-            if k in data or k == "_id":
-                raise Exception("Field {} is reserved.".format(k))
-            data[k] = str(v)
+        data['extra'] = kwargs
```

Comment:
Why are we no longer checking if it's reserved?

Reply:
I put everything that is extra into 'extra', so having something like `extra.hash` wouldn't cause name collision issues.
However, do you think that we should enforce not having something like `extra.hash` anyway?

Reply:
Gotcha. No, this makes sense. I'll resolve this.